assets/tests/api.test.ts

```diff
@@ -397,6 +402,50 @@ describe("fetchShapeForRoute", () => {
 describe("fetchFinishedDetour", () => {
   test("fetches missed stops in finished detour", () => {
     const stopData = stopDataFactory.buildList(3)
+    const [connection_stop_start, connection_stop_end] =
```

Comment:
**Nit (non-blocking)**: I goofed the last time I edited this, but it might be good to update the test name above to reflect that we care about more than just missed stops. I would almost say that the test should just be `test("fetches all relevant fields in finished detour", /* ... */)` or something like that.